torch/csrc/jit/script/module.h

```diff
@@ -460,6 +460,25 @@ struct Module {
 
   void save(const std::string& filename);
 
+  std::shared_ptr<Module> copy() const {
+    auto retval = std::make_shared<Module>();
+    std::unordered_map<at::Tensor*, at::Tensor*> parameter_remap;
```

Comment:
parameter remap not used - does it actually remap? :)

Reply:
this is from a previous stacked PR https://github.com/pytorch/pytorch/pull/15101. I can fix it there